src/apps/leads/src/store/leads.js

```diff
@@ -40,6 +41,7 @@ export function fetchLeads() {
             message: err.message,
           })
         );
+        throw err;
```

Comment:
Is this error handled somewhere? Otherwise this will cause some noise on Sentry.